gatsby-site/src/components/Navbar/navbar.css

```diff
@@ -71,9 +79,20 @@ span.navbar-toggler-icon {
 }
 
 .navbar.navbar-expand-md.navbar-light.bg-light {
-    background: rgba(0,0,0,0) !important;
-    padding-bottom: 0px;
-    padding-top: 0px;
-    width: 900px !important;
+    background: var(--background) !important;
+    margin: 0 auto;
+    width: 85% !important;
+    padding: 0;
+}
+
+@media only screen and (max-width: 815px) {    
+    .navbar.navbar-expand-md.navbar-light.bg-light {
+        width: 90% !important;
```

Comment:
is there a specific reason why you're using `!important` on a large percentage of your styles?

Reply:
To me, that makes me think the order of styles are being applied in the wrong order.

Reply:
reactstrap comes with default styling, `!important` overrides it - but I can see if there's places I've overused it